File: angular-tour-of-heroes/src/app/Entities/hero.ts
import {Universe} from 'src/app/Entities/universe';
import {HeroBio} from 'src/app/Entities/herobio';

export class Hero {
    heroId: number;
    heroName: string;
    powerLevel: string;
    pictureUrl: string;
    universe: Universe;
    bios: HeroBio[];
}

  export interface Heroes {
    Hero: Hero[];
}

File: Tour of Heroes API/Tour of Heroes/Tour of Heroes/Entities/Hero.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Tour_of_Heroes.Entities
{
    public class Hero
    {
        public int heroId { get; set; }
        public string heroName { get; set; }
        public string powerLevel { get; set; }
        public string pictureUrl { get; set; }
        public Universe universe { get; set; }
        public List<HeroBio> heroBio { get; set; }
    }
}
